lib/ansible/modules/network/nxos/nxos_linkagg.py

```diff
@@ -334,7 +334,7 @@ def parse_channel_options(module, output, channel):
     obj = {}
 
     group = channel['group']
-    obj['group'] = group
+    obj['group'] = str(group)
```

Comment:
Why is the value explicitly converted to string here?
As the option type is changed to `str` the type conversion is not required.

Reply:
This is a key that we fetched from the device, not the key from argspec. Some NXOS device returns int type, some returns str. So we want to make sure any group is str only to avoid idempotence issue.